app/src/main/java/ch/epfl/sweng/erpa/GameList.java

```diff
@@ -0,0 +1,14 @@
+package ch.epfl.sweng.erpa;
+
+import android.app.Activity;
+import android.support.v7.app.AppCompatActivity;
```

Comment:
We need to add a check for “import not used” check on the linter.

Useless imports have no runtime cost but pollutes the code, let's make an issue and fix them all in a single scoop !

Reply:
https://github.com/DivisionBy-Zero/erpa-sweng/issues/64